zebrad/src/components/mempool/crawler.rs

```diff
@@ -106,6 +119,36 @@ where
             transaction_ids.len()
         );
 
-        // TODO: Send transaction IDs to the download and verify stream (#2650)
+        if !transaction_ids.is_empty() {
+            if let Err(error) = self.queue_transactions(transaction_ids).await {
+                error!(
+                    "Failed to forward crawled transactions to mempool transaction downloader: {}",
+                    error
+                );
```

Comment:
I wasn't sure what level to use for this, but since I thought that this should in theory _never_ happen (unless the mempool service changes) but the crawler shouldn't stop if it does, I went with the highest error level.

This has an unfortunate side effect that the errors appear in the test output. I'm not sure if there's an easy way to disable this, but I was running tests with `RUST_LOG=no cargo test`.

Reply:
If we log an error during shutdown, it will really confuse users.

Instead, we should just silently exit (possibly with an info-level message):
https://github.com/ZcashFoundation/zebra/pull/2801#discussion_r719864562

Reply:
Done in [f668cde](https://github.com/jvff/zebra/commit/f668cdeaa64976e0d4484cb079e7015397148b28).